### PR TITLE
Adds lobsters and cuttlefish

### PR DESCRIPTION
**Pull request by Ketrai**
Adds cuttlefish, adds lobsters.

- orderable from cargo. (subject to change later, as we do have fishing mechanics) Crate costs 20 points.
- each crate comes with six lobsters and eight cuttlefish. Yes, this may seem like a lot, but if cargo wants to eat raw food with 5 nutrition each that's on them, might as well order pizza. even numbers for easy cooking
- Two new cookable dishes. cooked lobster for that luxury dinner, and cooked cuttlefish for those with an acquired taste.
- sprites original by me.

image

### CODE REVIEW

File: code/modules/food/food/snacks_vr.dm [hunk 1]
```diff
@@ -277,6 +277,57 @@
...
+	icon_state = "lobster_cooked"
+	trash = /obj/item/trash/plate
+	nutriment_amt = 20
+	nutriment_desc = list("lemon" = 2, "lobster" = 5, "salad" = 2)
+
+/obj/item/weapon/reagent_containers/food/snacks/lobstercooked/Initialize()
+	..()
+	bitesize = 5
+	reagents.add_reagent("protein", 20)
+	reagents.add_reagent("tricordrazine", 5)
```
Comment (Novacat) [inline code/modules/food/food/snacks_vr.dm]: The nutriment/protein amounts seem fine, but not really sure about having tricordrazine/iron. Tricordrazine is mainly a donk pocket/sin pocket thing, and Iron is mainly a liquidfood thing.

Comment (Ketrai) [reply code/modules/food/food/snacks_vr.dm]: Ah yeah, I forgot to describe that part. Intended to serve as something the chef can make for expiditions to virgo 4b. Though may need to increase the itemsize. Figured it'd be fine as it is more expensive/difficult to make than any ol food. you need to wait for cargo, and it costs cargo points.

But if you're that opposed, sure we can remove it.

Comment (Novacat) [reply code/modules/food/food/snacks_vr.dm]: Yeah, remove the tricord/iron. Nutriment and protein are plenty beneficial on its own medically, and I would rather not completely obsolete donk/sin pockets.

Comment (Ketrai) [reply code/modules/food/food/snacks_vr.dm]: except you can easily get donk pockets and sin pockets anywhere. Why does junkfood heal but classy food doesn't ? 📦

**Comment by Ketrai** [comment]
image

**Comment by Ketrai** [comment]
Been 5 days, doesn't sound like there's any objections.

**Pull request merged**